ppr-ui/src/composables/mhrRegistration/useNewMhrRegistration.ts

```diff
@@ -198,9 +200,44 @@ export const useNewMhrRegistration = () => {
   const fetchMhRegistrations = async (): Promise<void> => {
     const draftFilings = await getMhrDrafts()
     const myMhrHistory = await mhrRegistrationHistory()
-    setMhrTableHistory([...draftFilings, ...myMhrHistory])
+    const filteredMhrHistory = addHistoryDraftsToMhr(myMhrHistory, draftFilings)
+    setMhrTableHistory([...filteredMhrHistory])
   }
 
+  function addHistoryDraftsToMhr (mhrHistory: MhRegistrationSummaryIF[], mhrDrafts: MhrDraftTransferApiIF[]):
+    MhRegistrationSummaryIF[] {
+    const sorter = require('lodash')
+    const sortedDraftFilings = sorter.orderBy(mhrDrafts, ['createDateTime'], ['desc'])
+    // add drafts to Registrations..
+    const filteredMhrHistory = sorter
+      .orderBy(mhrHistory.filter(t => t.registrationDescription === 'REGISTER NEW UNIT'), ['createdDateTime'], ['desc'])
+    filteredMhrHistory.forEach(transfer => {
+      transfer.baseRegistrationNumber = transfer.mhrNumber
+      var mhrDrafts = sortedDraftFilings.filter(s => s.mhrNumber === transfer.mhrNumber)
+      if (mhrDrafts?.length > 0) {
+        transfer.hasDraft = true
+        transfer.changes = []
+        mhrDrafts.forEach(draft => {
```

Comment:
Some spacing might help for readability. 

Don't use `var` please, `let` if `mhrDrafts` but `const` otherwise. 
So in this case, `const` 

please use more accurate names in your filter expressions, `t` and `s` is not very intuitive to any other developer who might work here. 

Also, please check if there is an ENUM value for `REGISTER NEW UNIT`.

Reply:
Good Catch ...t,s  I put those there when I first... forgot to change..
Made some changes to clarify..